pkg/validate/validate.go

```diff
@@ -87,8 +87,8 @@ func All(ctx context.Context, validations ...BuildPath) error {
 	return nil
 }
 
-// BuildRunFields runs field validations against a BuildRun to detect
-// disallowed field combinations
+// BuildRun runs field validations against a BuildRun to detect
+// disallowed field combinations and issues
 func BuildRunFields(buildRun *build.BuildRun) (string, string) {
```

Comment:
Should not the comment start with the function name by convention ?

Reply:
Thanks. Fixed it.